design/design-file_organization.md

```diff
@@ -0,0 +1,93 @@
+- **Documentation** includes any documentation relevant to the project (link to Google Drive with kickoff notes, budget, process images, meeting notes, retros, etc.).
+- **Brand Design (optional)** contains any brand design work that Sparkbox has done.
+- **Wireframe / UX (optional)** includes a link to wireframes Sparkbox has created for this project or the actual files.
+
+
+## Root / -Projects / Project Name / -UI Explorations
+
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_76A2CDE662BC80FFEE513EA72F067BE0FAEA76B09B91F12E302C16DEA62A7533_1536344951200_Screen+Shot+2018-09-07+at+2.28.53+PM.png)
+
+
+Within the **UI Design** (and **Brand Design**) folders, files are named with a consistent pattern so that it is easy to know which is most current while still maintaining a project’s history. A new version of a file is added whenever there is a milestone in the project. For example, if the design direction changes, after a demo, etc.
+
+The goal is to no longer use an “Archive” tab within Sketch and instead use files to show the archived state. This allows us to use clean symbols and have smaller working files while still keeping a historical view of the project.
+
+Whoever is working on a particular file adds their initials at the end so that multiple designers can be working on the same project without overwriting each other.
+
+There is never a “final” file. Instead, the most recent numbered file is the most current file.
+
+
+## Root / -Projects / Project Name / -UI Iterations
+
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_76A2CDE662BC80FFEE513EA72F067BE0FAEA76B09B91F12E302C16DEA62A7533_1536345369339_Screen+Shot+2018-09-07+at+2.35.07+PM.png)
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_76A2CDE662BC80FFEE513EA72F067BE0FAEA76B09B91F12E302C16DEA62A7533_1536345369332_Screen+Shot+2018-09-07+at+2.35.45+PM.png)
+
+
+There are two ways the **-UI Iterations** folder could be structured. Every **-UI Iterations** folder will contain a **Symbol Library** file, which will serve as the source of truth for all symbols and consistent styles across the project. It can either have a list of versioned files that use that symbol library, or it can have subfolders like **-Foundry** or **-Newsletter** that represent specific parts of a larger project. These sub-folders should be used when the project is too large to contain in one Sketch file or to make it easier for multiple designers to work at the same time.
+
+
+## Root / Internal
+
+This folder holds all relevant assets that any Sparkbox designer might need access to or find helpful as they work on projects. Current internal projects live in this folder as well and are indexed with a **-**. Here’s an example of how this folder could be structured:
+
+- **Foundry Images**
+- **Brand**
+- **Bookshelf`
+- **Dribbble**
+- **Photos**
+- **Resources**
+- **Swag and Decor**
+
+
+Root / Internal / -Foundry Images
```

Comment:
Could we add a short line here that just says something like' "The **-Foundry Images** Folder contains the following subfolders" I think that's the point here, and I was confused for just a moment when I read it.

Reply:
So this is the convention used throughout this document. Are you suggesting we adjust all of the headings? I do think it's a bit confusing, but it's nice that it communicates the exact file path.